lisa/sut_orchestrator/azure/features.py

```diff
@@ -655,8 +655,9 @@ def get_raw_data_disks(self) -> List[str]:
         )
         matched = find_patterns_in_lines(cmd_result.stdout, [pattern])
         assert matched[0]
-        disk_array: List[str] = [""] * len(matched[0])
-        for disk in matched[0]:
+        matched_disk_array = set(matched[0])
```

Comment:
The commit message is confusing. The words "minor update", "new scenarios" have no information.

Reply:
updated